src/main/java/com/palominolabs/metrics/guice/intercept/CountedInterceptor.java

```diff
@@ -1,16 +1,16 @@
-package com.palominolabs.metrics.guice;
+package com.palominolabs.metrics.guice.intercept;
 
 import com.codahale.metrics.Counter;
 import com.codahale.metrics.annotation.Counted;
 import org.aopalliance.intercept.MethodInterceptor;
 import org.aopalliance.intercept.MethodInvocation;
 
-class CountedInterceptor implements MethodInterceptor {
+public class CountedInterceptor implements MethodInterceptor {
```

Comment:
Well, pretty much anything could hypothetically be re-used, and yet we don't make everything public... Can you go into more detail on what you're trying to achieve by making these public? Perhaps these classes are great as is, or maybe there are other/better ways to do it.